cluster/src/main/java/com/hartwig/pipeline/CommonArguments.java

```diff
@@ -73,6 +73,8 @@ public interface CommonArguments {
 
     Optional<String> pubsubTopicWorkflow();
 
+    Optional<String> pubsubTopicEnvironment();
```

Comment:
I know the precedent was there from before, but now that you've reworked the library I wonder if we need the "Topic" in the argument names?